hedera-node/src/main/java/com/hedera/services/state/expiry/NoopExpiringCreations.java

```diff
@@ -50,8 +49,14 @@ public ExpirableTxnRecord saveExpiringRecord(
 	}
 
 	@Override
-	public ExpirableTxnRecord.Builder buildExpiringRecord(long otherNonThresholdFees, ByteString hash,
-			TxnAccessor accessor, Instant consensusTime, TxnReceipt receipt, ServicesContext ctx) {
+	public ExpirableTxnRecord.Builder buildExpiringRecord(
+			long otherNonThresholdFees,
+			byte[] hash,
+			TxnAccessor accessor,
+			Instant consensusTime,
+			TxnReceipt receipt, 
+                        ServicesContext ctx
```

Comment:
Parameters aren't aligned here.

Reply:
Fixed in https://github.com/hashgraph/hedera-services/pull/1525